### PR TITLE
chore(review overview): hide buttons

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
Waiting on BE PR to be approved, so we have to hide buttons for now

## Solution
add `display="none"`

### CODE REVIEW

File: src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx [hunk 1]
```diff
@@ -339,10 +339,17 @@ export const RequestOverview = ({
...
       id: "actions",
       header: () => (
-        <Th borderBottom="1px solid" borderColor="gray.100" w="10rem" />
+        <Th
+          borderBottom="1px solid"
+          borderColor="gray.100"
+          w="10rem"
+          // NOTE: This line and the `display="none"` on the `HStack`
+          // should be removed once buttons are in on backend
+          display="none"
```
Comment (alexanderleegs) [inline src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx:348]: actually do we need the `display=none` here? the gray background cuts off atm, we just want the display none on cells right

Screenshot 2023-03-29 at 6 29 06 PM

**Review by alexanderleegs — APPROVED** [review]
small nits, but otherwise lgtm

File: src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx [hunk 2]
```diff
@@ -339,10 +339,17 @@ export const RequestOverview = ({
     columnHelper.display({
       id: "actions",
       header: () => (
-        <Th borderBottom="1px solid" borderColor="gray.100" w="10rem" />
+        <Th
+          borderBottom="1px solid"
+          borderColor="gray.100"
+          w="10rem"
+          // NOTE: This line and the `display="none"` on the `HStack`
+          // should be removed once buttons are in on backend
```
[the remark this answers is not in the document]
Comment (seaerchin) [reply src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx]: filed and changed!

File: src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx (hunk 1 above)
Comment (seaerchin) [reply src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx]: good catch, changed width to `0rem` to avoid this;  tfw u cant see good

**Pull request merged**